### DISCUSSION TITLE
RAV (Recursive Annotation Variation) feature

### DISCUSSION THREAD

**Discussion by sugizo** [niklasf/python-chess#1017] [Ideas]
nice to have RAV (Recursive Annotation Variation) feature too in python chess

https://php-chess.readthedocs.io/en/latest/read-rav/

thanks and best regards

**Comment by niklasf** [comment]
Variations are already supported by the parser and the [game model](https://python-chess.readthedocs.io/en/latest/pgn.html#game-model).
